accepted/00075-live-patching-with-clm.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: Live Patching with CLM
+- Start Date: 2021-02-08
+- RFC PR:
+
+# Summary
+[summary]: #summary
+
+Improve live patching workflow using CLM by enhancing user experience.
+
+# Motivation
+[motivation]: #motivation
+
+While it is possible to implement a live patching solution in Uyuni using Content Lifecycle Management features, it involves too much manual work and is not an easy thing to do for less experienced users.
+
+This feature is about enhancing the live patching experience in Uyuni by implementing additional CLM functionality to make the process easier for users.
+
+The existing live patching integration in Uyuni is described in the initial [Live Patching RFC](https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00027-sle-live-patching.md).
+
+# Detailed design
+[design]: #detailed-design
+
+## Live patching CLM workflow
+
+SLES channels can be prepared for live patching applications using the Content Lifecycle Management feature in Uyuni. This is done by adding a set of filters that exclude the regular kernel package updates and include live patch versions of those. However, the UI doesn't provide any special workflow for live patching and this process must be done manually by the user.
+
+Below are some examples of such sets of filters:
+
+**Example A:** Live patching project with a single filter to exclude regular kernel packages newer than a specific version
+ - `DENY erratum contains package with epoch/version/release greater than kernel-default <EVR> (package_nevr)`
+
+**Example B:** Live patching project using `ALLOW` overrides
+ - `DENY errata containing "update for the Linux Kernel" in synopsis`
+ - `ALLOW errata containing "Live Patch" in synopsis` *- an override to the first rule*
+ - `ALLOW a specific patch containing current kernel version (via advisory name)` *- an override to the first rule*
+
+**Example C:** Live patching project using reboot flag
+ - `DENY errata containing reboot_suggested (keyword)`
+ - `ALLOW errata containing "Live Patch" in synopsis` *- an override to the first rule*
+ - `ALLOW a specific patch containing current kernel version (via advisory name)` *- an override to the first rule*
+
+## CLM filter wizards
+
+The process of live patching can be made more user-friendly using "CLM filter wizards". The wizards are alternatives to the regular filter creation dialogs that provide a tailored interaction to create multiple filters that serve a single purpose. These dialogs can be accessed via the "Use wizard" button in the filter creation dialog. This concept can be extended to other use cases and workflows in the future.
+
+When completed, the wizards will create multiple filters that look like the examples mentioned in the section above to set up a project with a specific purpose. Since live patching channels can be composed using the existing filter types, no additional filter type needs to be implemented.
```

Comment:
We need to agree on what exact filter we want to setup. Having "options" in an RFC is ok for discussion, but the final version must describe 1 way we want to go. Otherwise the development team has again to make a decission.

Reply:
Since there weren't any concrete objections and as the most sensible choice at this stage, I went with Example A